packages/video_player/lib/video_player.dart

```diff
@@ -419,6 +432,33 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
     value = value.copyWith(volume: volume.clamp(0.0, 1.0));
     await _applyVolume();
   }
+
+  Future<void> _applySpeed() async {
+    if (!value.initialized || _isDisposed) {
+      return;
+    }
+
+    // On iOS setting the speed will start playing the video automatically
```

Comment:
I would include this in the documentation for `setSpeed()`.

Reply:
We don't automatically start playing the video when setting the speed. I've improved the comments to hopefully make this clearer.